roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- include: ./checks/check_system.yml
```

Comment:
Can you test if these work as a relative path without the ./ in front? Just a style thing.